python/ray/data/_internal/execution/streaming_executor.py

```diff
@@ -352,14 +352,18 @@ def _consumer_idling(self) -> bool:
         return len(self._output_node.outqueue) == 0
 
     def _report_current_usage(self) -> None:
-        cur_usage = self._resource_manager.get_global_usage()
+        running_usage = self._resource_manager.get_global_running_usage()
+        pending_usage = self._resource_manager.get_global_pending_usage()
         limits = self._resource_manager.get_global_limits()
         resources_status = (
             "Running: "
-            f"{cur_usage.cpu:.4g}/{limits.cpu:.4g} CPU, "
-            f"{cur_usage.gpu:.4g}/{limits.gpu:.4g} GPU, "
-            f"{cur_usage.object_store_memory_str()}/"
-            f"{limits.object_store_memory_str()} object_store_memory"
+            f"{running_usage.cpu:.4g}/{limits.cpu:.4g} CPU, "
+            f"{running_usage.gpu:.4g}/{limits.gpu:.4g} GPU, "
+            f"{running_usage.object_store_memory_str()}/"
+            f"{limits.object_store_memory_str()} object_store_memory; "
+            "Pending actors: "
+            f"{pending_usage.cpu:.4g} on CPU, "
+            f"{pending_usage.gpu:.4g} on GPU"
```

Comment:
nit: to reduce clutter in the progress output, and keep consistent with the running usages.
```suggestion
            "Pending: "
            f"{pending_usage.cpu:.4g} CPU, "
            f"{pending_usage.gpu:.4g} GPU"
```

Reply:
Also, i think we should re-order this, so that the pening usage comes directly after the current running usage, and before the object store memory. Since the current + pending usages are related.

Reply:
Thank you Scott, reordering makes sense. My original intention to paraphrase the summary is to make it more clear that there are tasks or actors executing on CPUs and GPUs and pending actors scheduled on CPU/GPU. The existing "Running: 0/8 CPU, 1/1 GPU" phrase is quite clear about how many resources are currently used. IMO, "Pending: 0 CPU, 1 GPU" isn't as clear as "Pending actors: 0 on CPU, 1 on GPU" which indicates that 0 actors are scheduled on CPU, and 1 actor is scheduled on GPU. I might be overly paranoid though.

Reply:
Actually, Hao suggested including [obj_store_mem_pending_task_outputs](https://github.com/ray-project/ray/blob/82af058f3456dd7088b94a8d29a95450f73d215b/python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py#L338) in pending usage which I planned to address in a future PR. If we were to introduce `obj_store_mem_pending_task_outputs`, the summary line will look something like "Running: 0/8 CPU, 1/1 GPU, 48.0B/46GB object_store_memory; Pending: 0 CPU, 1 GPU, 42B object_store_memory". Reordering will not make the most sense to me as we might need to reorder it back to the format above. Also, the current object_store_memory indicated in the summary line corresponds to the running usage, and I think it is reasonable for it coming after "Running". Let me know if this makes sense to you 😄

Reply:
Tl;dr for my recent commit: I followed the wordings of running usages for pending usages but didn't change the ordering.

Reply:
> Actually, Hao suggested including [obj_store_mem_pending_task_outputs](https://github.com/ray-project/ray/blob/82af058f3456dd7088b94a8d29a95450f73d215b/python/ray/data/_internal/execution/interfaces/op_runtime_metrics.py#L338) in pending usage which I planned to address in a future PR. If we were to introduce `obj_store_mem_pending_task_outputs`, the summary line will look something like "Running: 0/8 CPU, 1/1 GPU, 48.0B/46GB object_store_memory; Pending: 0 CPU, 1 GPU, 42B object_store_memory". Reordering will not make the most sense to me as we might need to reorder it back to the format above. Also, the current object_store_memory indicated in the summary line corresponds to the running usage, and I think it is reasonable for it coming after "Running". Let me know if this makes sense to you 😄

Got it, I agree, we can keep it as you have in your most recent commit